src/main/java/com/demo/controller/BaseController.java

```diff
@@ -23,6 +26,7 @@ public String root() {
 	@RequestMapping("/index")
 	public String index(Model model){
         log.info("index");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "index" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1192)